test/functional/api/cas/cache_config.py

```diff
@@ -263,6 +264,114 @@ def default_nhit_params():
         return nhit_params
 
 
+# Specify layout of metadata on SSD
+class MetadataLayout(Enum):
+    striping = 0
+    sequential = 1
+    DEFAULT = striping
+
+
+# Specify how IO requests unaligned to 4KiB should be handled
+class UnalignedIo(Enum):
+    PT = 0      # use PT mode
+    cache = 1   # use current cache mode
+    DEFAULT = cache
+
+
+# Specify if IO scheduler will be used when handling IO requests
+class UseIoScheduler(Enum):
+    off = 0
+    on = 1
+    DEFAULT = on
+
+
+class KernelParameters:
+    seq_cut_off_mb_DEFAULT = 1
+    max_writeback_queue_size_DEFAULT = 65536
+    writeback_queue_unblock_size_DEFAULT = 60000
+
+    def __init__(
+            self,
+            metadata_layout: MetadataLayout = None,
+            unaligned_io: UnalignedIo = None,
+            use_io_scheduler: UseIoScheduler = None,
+            seq_cut_off_mb: int = None,
+            max_writeback_queue_size: int = None,
+            writeback_queue_unblock_size: int = None
+    ):
+        self.metadata_layout = metadata_layout
+        self.unaligned_io = unaligned_io
+        self.use_io_scheduler = use_io_scheduler
+        # Specify default sequential cut off threshold value in MiB
+        # 0 - sequential cut off disabled, 1 or larger = sequential cut off threshold (default is 1)
+        self.seq_cut_off_mb = seq_cut_off_mb
+        # Specify optimal write queue size, default - 65536
+        self.max_writeback_queue_size = max_writeback_queue_size
+        # Specify unblock threshold for write queue, default - 60000
+        self.writeback_queue_unblock_size = writeback_queue_unblock_size
+
+    def __eq__(self, other):
+        return (
+            equal_or_default(self.metadata_layout, other.metadata_layout, MetadataLayout.DEFAULT)
+            and equal_or_default(self.unaligned_io, other.unaligned_io, UnalignedIo.DEFAULT)
+            and equal_or_default(
+                self.use_io_scheduler, other.use_io_scheduler, UseIoScheduler.DEFAULT
+            )
+            and equal_or_default(
+                self.seq_cut_off_mb, other.seq_cut_off_mb,
+                self.seq_cut_off_mb_DEFAULT
+            )
+            and equal_or_default(
+                self.max_writeback_queue_size, other.max_writeback_queue_size,
+                self.max_writeback_queue_size_DEFAULT
+            )
+            and equal_or_default(
+                self.writeback_queue_unblock_size, other.writeback_queue_unblock_size,
+                self.writeback_queue_unblock_size_DEFAULT
+            )
+        )
+
+    @classmethod
+    def DEFAULT(cls):
```

Comment:
Instead of having this method we can set default values for constructor arguments.

Reply:
Default values for constructor are None, because that reflects the default action when you load the kernel module with no parameters provided. We need DEFAULT() for reference, but we don't want to explicitly set parameters by default.